ADMIN.adoc

```diff
@@ -0,0 +1,35 @@
+= Repository Administrator Responsibilities
+
+== Administrator rotation schedule:
+
+Every 2 weeks a different project member assumes the role of repository administrator.
+Signing up to be administrator is voluntary.
+
+== Administrator responsibilities:
+
+As the administrator, you are in charge of performing the following duties:
+
+=== Being the Point of Contact for general issues related to the Supplementary Style Guide repository
+
+* You are the point of contact in case other contributors experience issues with the repository.
```

Comment:
```suggestion
* You are the point of contact if contributors experience issues with the repository.
```

Reply:
Being pedantic now rather than someone pointing out these minor issues down the line.